RubeesEat/wwwroot/css/UserHomePage.css

```diff
@@ -0,0 +1,112 @@
+.positive-balance {
```

Comment:
This CSS added in this file was not part of the issue. We don't want to change the content styling in this issue. As such, we are going to revert these changes. I would suggest starting from scratch and styling the page again as this has a few issues:

- Too many selectors reference specific elements on the page (`#list`, `#hello`, etc.). We should try and make reusable components that we can use on multiple pages.
- There are a lot of "need spacing here" changes. Ideally, we don't need to apply many margin/paddings to components individually as they should have good defaults
- Too many font size changes that should also be handled on a global scale. We don't want the sizes to be all over the place so there should be the same sizes regardless of which page the user is viewing.